### PR TITLE
chore(curriculum): update lab orbit and earlier asserts

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

I updated a few of the tests that only had one or two tests that need altered. 

Additionally I modified the [keyframes orbit rule test](https://github.com/a2937/freeCodeCamp/blob/96c9d4c46d95b09bc963bde7a11078e98428b112/curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md#L214) as it seemed to have redundant data.

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-favorite-icon-toggler/66bf6bacf178eac7b96d4f5e.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Initially, the `span` elements should contain the code `&#9825;` to represent an
 
 ```js
 const inputs = document.querySelectorAll('ul li span.favorite-icon');
-assert(inputs.length)
+assert.notEmpty(inputs)
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-favorite-icon-toggler/66bf6bacf178eac7b96d4f5e.md:61]:
```suggestion
assert.isNotEmpty(inputs)
```
testing is recommented

File: curriculum/challenges/english/25-front-end-development/lab-favorite-icon-toggler/66bf6bacf178eac7b96d4f5e.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ When the `span` element is clicked, and it contains the class `filled`, you shou
 
 ```js
 const spanElements = document.querySelectorAll('.favorite-icon');
-assert(spanElements.length);
+assert.notEmpty(spanElements);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-favorite-icon-toggler/66bf6bacf178eac7b96d4f5e.md:72]:
```suggestion
assert.isNotEmpty(spanElements);
```

File: curriculum/challenges/english/25-front-end-development/lab-favorite-icon-toggler/66bf6bacf178eac7b96d4f5e.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ When the `span` element is clicked, and it doesn't contain the class `filled`, y
...
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-favorite-icon-toggler/66bf6bacf178eac7b96d4f5e.md:88]:
```suggestion
assert.isNotEmpty(spanElements);
```

File: curriculum/challenges/english/25-front-end-development/lab-lightbox-viewer/66db57ad34c7089b9b41bfd6.md [hunk 4]
```diff
@@ -142,7 +142,7 @@ assert.equal(new __helpers.CSSHelp(document).getStyle('.lightbox')?.left, '0px')
 Your `.lightbox` element should have a background color.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.lightbox')?.backgroundColor);
+assert.exists(new __helpers.CSSHelp(document).getStyle('.lightbox')?.backgroundColor);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-lightbox-viewer/66db57ad34c7089b9b41bfd6.md:145]:
```suggestion
assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('.lightbox')?.backgroundColor);
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 5]
```diff
@@ -50,19 +50,19 @@ The contents of the `body` element should be centered on the page by setting the
 
 ```js
 
-assert(new __helpers.CSSHelp(document).getStyle('body')?.height === '100vh' && new __helpers.CSSHelp(document).getStyle('body')?.display === 'flex' && new __helpers.CSSHelp(document).getStyle('body')?.justifyContent === 'center'&& new __helpers.CSSHelp(document).getStyle('body')?.alignItems === 'center');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('body')?.height === '100vh' && new __helpers.CSSHelp(document).getStyle('body')?.display === 'flex' && new __helpers.CSSHelp(document).getStyle('body')?.justifyContent === 'center'&& new __helpers.CSSHelp(document).getStyle('body')?.alignItems,'center');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:53]: uhhhh you should really divide this in 4 `strictEqual`

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 6]
```diff
@@ -50,19 +50,19 @@ The contents of the `body` element should be centered on the page by setting the
...
 ```js
-assert(document.querySelector('div')?.getAttribute('class') === 'space');
+assert.strictEqual(document.querySelector('div')?.getAttribute('class'),'space');
 ```
 
 The `space` `div` should have a width and height of `200px`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.space')?.width === '200px' && new __helpers.CSSHelp(document).getStyle('.space')?.height === '200px');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.space')?.width === '200px' && new __helpers.CSSHelp(document).getStyle('.space')?.height, '200px');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:65]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.space')?.width === '200px' && new __helpers.CSSHelp(document).getStyle('.space')?.height, '200px');
```
this must be divided in two

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 7]
```diff
@@ -74,34 +74,35 @@ assert.exists(document.querySelector('div.space div.earth'));
...
-assert(document.querySelector('.space > div')?.getAttribute('class') === 'earth');
+assert.strictEqual(document.querySelector('.space > div')?.getAttribute('class') , 'earth');
 ```
 
 The `earth` `div` should have a width and height of `100px`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.earth')?.width === '100px' && new __helpers.CSSHelp(document).getStyle('.earth')?.height === '100px');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.width , '100px');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.width === '100px' && new __helpers.CSSHelp(document).getStyle('.earth')?.height, '100px');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:84]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.height, '100px');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 8]
```diff
@@ -74,34 +74,35 @@ assert.exists(document.querySelector('div.space div.earth'));
...
-assert(new __helpers.CSSHelp(document).getStyle('.earth')?.position === 'absolute');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.position , 'absolute');
 ```
 
 The `earth` `div` should have a `top` and `left` position of `50%`.
 
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.earth')?.top === '50%' && new __helpers.CSSHelp(document).getStyle('.earth')?.left === '50%');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.top === '50%' && new __helpers.CSSHelp(document).getStyle('.earth')?.left , '50%');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:98]: must be divided

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 9]
```diff
@@ -113,25 +114,25 @@ assert.exists(document.querySelector('div.space div.orbit'));
...
 ```js
-assert(document.querySelector('.earth')?.nextElementSibling?.classList?.contains('orbit'));
+assert.isTrue(document.querySelector('.earth')?.nextElementSibling?.classList?.contains('orbit'));
 ```
 
 The `orbit` `div` should have a width and height of `200px`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.orbit')?.width === '200px' && new __helpers.CSSHelp(document).getStyle('.orbit')?.height === '200px');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.orbit')?.width === '200px' && new __helpers.CSSHelp(document).getStyle('.orbit')?.height,'200px');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:123]: must be divided

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 10]
```diff
@@ -143,112 +144,110 @@ assert.exists(document.querySelector('div.space div.orbit div.moon'));
 The `moon` `div` should have a width and height of `30px`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.moon')?.width === '30px' && new __helpers.CSSHelp(document).getStyle('.moon')?.height === '30px');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.width === '30px' && new __helpers.CSSHelp(document).getStyle('.moon')?.height, '30px');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:147]: must be divided

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 11]
```diff
@@ -143,112 +144,110 @@ assert.exists(document.querySelector('div.space div.orbit div.moon'));
...
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.moon')?.transform === 'translateX(-50%)');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.transform , 'translateX(-50%)');
 ```
 
 Your `.earth` `div` element should have a background color.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.earth')?.backgroundColor);
+assert.exists(new __helpers.CSSHelp(document).getStyle('.earth')?.backgroundColor);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:177]:
```suggestion
assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('.earth')?.backgroundColor);
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 12]
```diff
@@ -143,112 +144,110 @@ assert.exists(document.querySelector('div.space div.orbit div.moon'));
...
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.earth')?.borderRadius === '50%');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.borderRadius, '50%');
 ```
 
 Your `.moon` `div` element should have a background color.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.moon')?.backgroundColor);
+assert.exists(new __helpers.CSSHelp(document).getStyle('.moon')?.backgroundColor);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:189]:
```suggestion
assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('.moon')?.backgroundColor);
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/25-front-end-development/lab-favorite-icon-toggler/66bf6bacf178eac7b96d4f5e.md (hunk 1 above)
Comment (a2937) [reply curriculum/challenges/english/25-front-end-development/lab-favorite-icon-toggler/66bf6bacf178eac7b96d4f5e.md:61]: Dang it. I really should have stuck to one file at a time.

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 13]
```diff
@@ -74,34 +78,36 @@ assert.exists(document.querySelector('div.space div.earth'));
 The `earth` div should be the first child of the `space` `div`.
 
 ```js
-assert(document.querySelector('.space > div')?.getAttribute('class') === 'earth');
+assert.strictEqual(document.querySelector('.space > div')?.getAttribute('class') , 'earth');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:81]:
```suggestion
assert.strictEqual(document.querySelector('.space > div')?.getAttribute('class'), 'earth');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 14]
```diff
@@ -74,34 +78,36 @@ assert.exists(document.querySelector('div.space div.earth'));
...
 ```
 
 The `earth` `div` should have a width and height of `100px`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.earth')?.width === '100px' && new __helpers.CSSHelp(document).getStyle('.earth')?.height === '100px');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.width , '100px');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:87]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.width, '100px');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 15]
```diff
@@ -74,34 +78,36 @@ assert.exists(document.querySelector('div.space div.earth'));
...
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.height, '100px');
 ```
 
 The `earth` `div` should use `absolute` positioning.
 
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.earth')?.position === 'absolute');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.position , 'absolute');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:95]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.position, 'absolute');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 16]
```diff
@@ -74,34 +78,36 @@ assert.exists(document.querySelector('div.space div.earth'));
...
 ```
 
 The `earth` `div` should have a `top` and `left` position of `50%`.
 
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.earth')?.top === '50%' && new __helpers.CSSHelp(document).getStyle('.earth')?.left === '50%');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.top, '50%') 
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.left , '50%');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:103]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.left, '50%');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 17]
```diff
@@ -74,34 +78,36 @@ assert.exists(document.querySelector('div.space div.earth'));
...
 ```
 
 The `earth` `div` should be moved back and up by `50%` by setting the `transform` property to `translate(-50%, -50%)`.
 
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.earth')?.transform === 'translate(-50%, -50%)');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.transform ,'translate(-50%, -50%)');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:110]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.earth')?.transform,'translate(-50%, -50%)');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 18]
```diff
@@ -113,25 +119,26 @@ assert.exists(document.querySelector('div.space div.orbit'));
...
-assert(document.querySelector('.earth')?.nextElementSibling?.classList?.contains('orbit'));
+assert.isTrue(document.querySelector('.earth')?.nextElementSibling?.classList?.contains('orbit'));
 ```
 
 The `orbit` `div` should have a width and height of `200px`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.orbit')?.width === '200px' && new __helpers.CSSHelp(document).getStyle('.orbit')?.height === '200px');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.orbit')?.width,'200px')
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.orbit')?.height,'200px');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:129]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.orbit')?.width, '200px')
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.orbit')?.height, '200px');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 19]
```diff
@@ -113,25 +119,26 @@ assert.exists(document.querySelector('div.space div.orbit'));
...
 ```
 
 The `orbit` `div` should use `absolute` positioning.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.orbit')?.position === 'absolute');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.orbit')?.position , 'absolute');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:135]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.orbit')?.position, 'absolute');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 20]
```diff
@@ -113,25 +119,26 @@ assert.exists(document.querySelector('div.space div.orbit'));
...
 ```
 
 The `orbit` `div` should be positioned at the center of the `space` `div` using a `transform` property that shifts it by `-50%` on both the vertical and horizontal axes.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.orbit')?.transform === 'translate(-50%, -50%)');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.orbit')?.transform , 'translate(-50%, -50%)');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:141]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.orbit')?.transform, 'translate(-50%, -50%)');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 21]
```diff
@@ -143,112 +150,111 @@ assert.exists(document.querySelector('div.space div.orbit div.moon'));
...
-assert(new __helpers.CSSHelp(document).getStyle('.moon')?.width === '30px' && new __helpers.CSSHelp(document).getStyle('.moon')?.height === '30px');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.width, '30px') 
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.height, '30px');
 ```
 
 The `moon` `div` should be positioned using `absolute` positioning.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.moon')?.position === 'absolute');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.position , 'absolute');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:160]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.position, 'absolute');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 22]
```diff
@@ -143,112 +150,111 @@ assert.exists(document.querySelector('div.space div.orbit div.moon'));
...
 ```
 
 The top edge of the `.moon` element should be aligned with the top edge of its containing block.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.moon')?.top === '0px');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.top,'0px');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:166]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.top, '0px');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 23]
```diff
@@ -143,112 +150,111 @@ assert.exists(document.querySelector('div.space div.orbit div.moon'));
...
 ```
 
 You should move the left edge of the `.moon` element to the center of the containing block (`50%` from the left side).
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.moon')?.left === '50%');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.left , '50%');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:172]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.left, '50%');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 24]
```diff
@@ -143,112 +150,111 @@ assert.exists(document.querySelector('div.space div.orbit div.moon'));
...
 ```
 
 The `.moon` `div` should be adjusted horizontally by translating it to `-50%` across the x-axis.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.moon')?.transform === 'translateX(-50%)');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.transform , 'translateX(-50%)');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:178]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.transform, 'translateX(-50%)');
```

File: curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md [hunk 25]
```diff
@@ -143,112 +150,111 @@ assert.exists(document.querySelector('div.space div.orbit div.moon'));
...
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.moon')?.backgroundColor);
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('.moon')?.backgroundColor);
 ```
 
 Your `.moon` element should have `border-radius` of `50%` to make it look like a circle.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.moon')?.borderRadius === '50%');
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.borderRadius , '50%');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md:202]:
```suggestion
assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.borderRadius, '50%');
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
linting

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
last one I think

**Review by majestic-owl448 — APPROVED** [review]
Thank you for the work

**Review by moT01 — APPROVED** [review]

**Pull request merged**